static/js/components/ShiftSummary.jsx

```diff
@@ -5,20 +5,16 @@ import dayjs from "dayjs";
 // eslint-disable-next-line import/no-unresolved
 import Form from "@rjsf/mui";
 import validator from "@rjsf/validator-ajv8";
-import {
-  Collapse,
-  Divider,
-  List,
-  ListItem,
-  Paper,
-  Tooltip,
-} from "@mui/material";
+import Collapse from "@mui/material/Collapse";
+import Divider from "@mui/material/Divider";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import Paper from "@mui/material/Paper";
+import Tooltip from "@mui/material/Tooltip";
 import { showNotification } from "baselayer/components/Notifications";
-import {
-  ExpandLess,
-  ExpandMore,
-  HelpOutlineOutlined,
-} from "@mui/icons-material";
+import ExpandLess from "@mui/icons-material/ExpandLess";
```

Comment:
The import order is incorrect. All the MUI or npm installable stuff should come before the baselayer and skyportal imports. Just like in python

Reply:
Maybe it was already incorrect, in which case please fix it :)